### PR TITLE
[TR] Migration sf 2.7

### PR DESCRIPTION
**Pull request by skeleton**

### CODE REVIEW

**Comment by jjanvier** [comment]
GTM

File: src/Oro/Bundle/FilterBundle/Tests/Unit/Form/Type/AbstractTypeTestCase.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ protected function createMockOptionsResolver()
     }
 
     /**
-     * @dataProvider setDefaultOptionsDataProvider
+     * @dataProvider configureOptionsDataProvider
```
Comment (phaseinducer) [inline src/Oro/Bundle/FilterBundle/Tests/Unit/Form/Type/AbstractTypeTestCase.php:89]: no blank line ?

File: src/Oro/Bundle/FormBundle/Form/Type/EntityIdentifierType.php [hunk 2]
```diff
@@ -83,63 +83,53 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
...
             )
         )
-        ->setAllowedValues(
-            array(
-                'multiple' => array(true, false),
-            )
-        );
-        $resolver->setRequired(array('class'));
+        ->setAllowedValues('multiple', [true, false])
+        ->setRequired(array('class'));
```
Comment (phaseinducer) [inline src/Oro/Bundle/FormBundle/Form/Type/EntityIdentifierType.php:87]: []

File: src/Oro/Bundle/FormBundle/Form/Type/OroDateType.php [hunk 3]
```diff
@@ -27,15 +27,36 @@ public function finishView(FormView $view, FormInterface $form, array $options)
...
-                'widget'         => 'single_text',
-                'placeholder'    => 'oro.form.click_here_to_select',
-            )
-        );
+            [
+                'format'      => 'yyyy-MM-dd',
+                'widget'      => 'single_text',
+                'placeholder' => 'oro.form.click_here_to_select',
+            ]
+        )->setNormalizer('placeholder', function (Options $options, $placeholder) use ($placeholderDefault) {
```
Comment (phaseinducer) [inline src/Oro/Bundle/FormBundle/Form/Type/OroDateType.php:42]: on a new line ?

File: src/Oro/Bundle/FormBundle/Form/Type/OroDateType.php [hunk 4]
```diff
@@ -27,15 +27,36 @@ public function finishView(FormView $view, FormInterface $form, array $options)
...
+            if (is_string($placeholder)) {
+                return $placeholder;
+            } elseif (is_array($placeholder)) {
+                $default = $placeholderDefault($options);
+
+                return array_merge(
+                    array('year' => $default, 'month' => $default, 'day' => $default),
```
Comment (phaseinducer) [inline src/Oro/Bundle/FormBundle/Form/Type/OroDateType.php:49]: []

File: src/Oro/Bundle/FormBundle/Form/Type/OroDateType.php [hunk 5]
```diff
@@ -27,15 +27,36 @@ public function finishView(FormView $view, FormInterface $form, array $options)
...
+                    $placeholder
+                );
+            }
+
+            return array(
```
Comment (phaseinducer) [inline src/Oro/Bundle/FormBundle/Form/Type/OroDateType.php:54]: []

File: src/Oro/Bundle/NavigationBundle/Menu/NavigationItemBuilder.php [hunk 6]
```diff
@@ -14,9 +14,9 @@
 class NavigationItemBuilder implements BuilderInterface
 {
     /**
-     * @var SecurityContextInterface
+     * @var TokenStorageInterface
```
Comment (phaseinducer) [inline src/Oro/Bundle/NavigationBundle/Menu/NavigationItemBuilder.php]: in one line

File: src/Oro/Bundle/NavigationBundle/Tests/Unit/Entity/Builder/HistoryItemBuilderTest.php [hunk 7]
```diff
@@ -13,9 +13,9 @@ class HistoryItemBuilderTest extends \PHPUnit_Framework_TestCase
     protected $em;
 
     /**
-     * @var \Symfony\Component\Security\Core\SecurityContextInterface
+     * @var \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface
```
Comment (phaseinducer) [inline src/Oro/Bundle/NavigationBundle/Tests/Unit/Entity/Builder/HistoryItemBuilderTest.php:16]: import class please

File: src/Oro/Bundle/NavigationBundle/Tests/Unit/Event/ResponseHistoryListenerTest.php [hunk 8]
```diff
@@ -18,9 +18,9 @@ class ResponseHistoryListenerTest extends \PHPUnit_Framework_TestCase
     protected $em;
 
     /**
-     * @var \Symfony\Component\Security\Core\SecurityContextInterface
+     * @var \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface
```
Comment (phaseinducer) [inline src/Oro/Bundle/NavigationBundle/Tests/Unit/Event/ResponseHistoryListenerTest.php:21]: import class

File: src/Oro/Bundle/NavigationBundle/Tests/Unit/Event/ResponseHistoryListenerTest.php [hunk 9]
```diff
@@ -226,13 +226,13 @@ public function getTitleService()
 
     /**
      * @param  \Oro\Bundle\NavigationBundle\Entity\Builder\ItemFactory   $factory
-     * @param  \Symfony\Component\Security\Core\SecurityContextInterface $securityContext
+     * @param  \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface $tokenStorage
```
Comment (phaseinducer) [inline src/Oro/Bundle/NavigationBundle/Tests/Unit/Event/ResponseHistoryListenerTest.php:229]: import classes please

File: src/Oro/Bundle/UserBundle/Security/WsseUserProvider.php [hunk 10]
```diff
@@ -24,18 +24,27 @@ class WsseUserProvider extends Provider
...
         PasswordEncoderInterface $encoder,
         Cache $nonceCache,
-        $lifetime = 300
-    ) {
-        parent::__construct($userProvider, $encoder, $nonceCache, $lifetime);
+        $lifetime=300,
+        $dateFormat='/^([\+-]?\d{4}(?!\d{2}\b))((-?)((0[1-9]|1[0-2])(\3([12]\d|0[1-9]|3[01]))?|W([0-4]\d|5[0-2])(-?[1-7])?|(00[1-9]|0[1-9]\d|[12]\d{2}|3([0-5]\d|6[1-6])))([T\s]((([01]\d|2[0-3])((:?)[0-5]\d)?|24\:?00)([\.,]\d+(?!:))?)?(\17[0-5]\d([\.,]\d+)?)?([zZ]|([\+-])([01]\d|2[0-3]):?([0-5]\d)?)?)?)?$/'
+    )
+    {
+        if(empty($providerKey))
```
Comment (phaseinducer) [inline src/Oro/Bundle/UserBundle/Security/WsseUserProvider.php]: wrong style

**Pull request closed without merging**